themes/socialblue/components/03-molecules/dropdown/dropdown--sky.scss

```diff
@@ -0,0 +1,17 @@
+// Behaviors arrow on the dropdown menu
+
+.socialblue--sky {
+  .dropdown.open .caret,
+  .enroll-action-form.open .caret,
+  .hero-footer__cta .open .caret,
+  .social-gtranslate-list--desktop.open > a:after {
+    transform: rotate(180deg);
+  }
```

Comment:
I think this is a change we can make in both flavours.

Additionally the following can make it look at bit more friendly:
```
.dropdown .caret {
    transition: transform 0.1s;
}
```

Reply:
@Kingdutch, I've moved these styles in the SocialBase theme and add `transition`.

Reply:
It was fixed in PR #1680